src/numeric.rs

```diff
@@ -188,7 +189,8 @@ impl Interval {
     ///  \end{cases}
     /// $$
     ///
-    /// The exact value is returned.
+    /// The exact value is returned.  Note that, as mandated by
+    /// IEEE-1788, when $a$ is ±0., `0.` is returned.
```

Comment:
Ditto:

```
When $b = 0$, `+0.0` is returned.
```

Reply:
I changed a bit the wording as `+0.0` is not valid Rust code.